### ISSUE TITLE
apt-get upgrade error - Illegal number: PARENT_PID

### ISSUE THREAD

**Issue by artooro** [jitsi/jigasi#6]
Today I did an apt-get dist-upgrade to update our jitsi-meet installation and got the following errors for jigasi.

```
Stopping jigasi: /etc/init.d/jigasi: 42: kill: Illegal number: PARENT_PID
invoke-rc.d: initscript jigasi, action "stop" failed.
dpkg: warning: subprocess old pre-removal script returned error exit status 2
dpkg: trying script from the new package instead ...
Stopping jigasi: /etc/init.d/jigasi: 42: kill: Illegal number: PARENT_PID
invoke-rc.d: initscript jigasi, action "stop" failed.
dpkg: error processing archive /var/cache/apt/archives/jigasi_1.0-58_amd64.deb (--unpack):
 subprocess new pre-removal script returned error exit status 2
insserv: warning: script 'K01restund' missing LSB tags and overrides
insserv: warning: script 'restund' missing LSB tags and overrides
jigasi seems to be already running, we found pidfile /var/run/jigasi.pid.
invoke-rc.d: initscript jigasi, action "start" failed.
dpkg: error while cleaning up:
 subprocess installed post-installation script returned error exit status 1
```

```
Errors were encountered while processing:
 /var/cache/apt/archives/jigasi_1.0-58_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

It seems like the process is not running but the PID file is still there. So I'm thinking if I delete the PID file that will resolve the issue. But just wanted to report this.

**Comment by artooro** [comment]
I did ```rm /var/run/jigasi.pid``` and then ```apt-get upgrade``` and it fixed the initial issue, but now also seeing:

```
find: `/var/log/jitsi/jigasi': No such file or directory
dpkg: error processing package jigasi (--configure):
 subprocess installed post-installation script returned error exit status 1
```

**Comment by artooro** [comment]
To fix that issue I did the following:

```
mkdir /var/log/jitsi/jigasi
chown jigasi:jitsi /var/log/jitsi/jigasi
apt-get upgrade
```

And there were no more errors and the service seemed to start OK.

**Comment by yanas** [comment]
This has been fixed with commit 75b997ede2.
